resources/views/layouts_new/authBase.blade.php

```diff
@@ -65,21 +65,34 @@
 			<p>{{ Lang::get('messages.notifications.available_in_en') }} <a href="{{ route('profile.language.set', ['en']) }}" class="alert-link">{{ Lang::get('messages.languages.en') }}</a>!</p>
 			@endif
 		</div>
-	</div>
-	
-	
+
+
+		@if (isset($modAlerts["approval"]) AND $modAlerts["approval"])
```

Comment:
So the message is shown if only `$modAlerts["approval"]` is set and is not empty?
Maybe consider using `@if (isset($modAlerts["approval"]) == true AND empty($modAlerts["approval"]) == true)`